oojs.md

```diff
@@ -24,13 +39,21 @@ Create an `Album` class that inherits from `Playlist`. It should also...
 - Have an `artist` (String) property that is determined by some input
 
 ```js
-Your answer goes here
+var inheritsFrom = function (child, parent) {
+    child.prototype = Object.create(parent.prototype);
+};
+class Album {
+  constructor(artist){
+    this.artist = artist;
+  }
+}
+inheritsFrom(Album, Playlist);
 ```
```

Comment:
That's cool, it's an old-school way to do that though and if you're already using ES6 `classes` you can and should just use `extends`.